.github/workflows/dependency-review.yml

```diff
@@ -0,0 +1,12 @@
+permissions: read-all
```

Comment:
The workflow is missing name and trigger.

Reply:
Updated.